waspc/data/Generator/templates/sdk/package.json

```diff
@@ -89,9 +89,8 @@
       "./crud/*": "./dist/crud/*.js",
       {=!  Used by our code, uncodumented (but accessible) for users. =}
       "./server/crud/*": "./dist/server/crud/*",
-      "./email": "./dist/email/index.js",
       {=!  Used by our code, uncodumented (but accessible) for users. =}
-      "./email/core/types": "./dist/email/core/types.js",
+      "./server/email/core/types": "./dist/server/email/core/types.js",
```

Comment:
If you know we still need these, it is maybe best if you move it below to section "still needed, reconsider during refactoring."? No need to do it now, just idea for future PRs.